test/e2e/alertmanager_test.go

```diff
@@ -28,86 +28,89 @@ import (
 )
 
 func TestAlertmanagerCreateDeleteCluster(t *testing.T) {
+	t.Parallel()
+
+	ctx := testFramework.NewTestCtx(t)
```

Comment:
I think `NewTestCtx` should be a method to the `framework` we are using elsewhere. 
This way you don't have to pass in `framework.KubeClient` if you want this context to create a non-namespaced object.